conductor/src/main/java/com/bluelinelabs/conductor/Controller.java

```diff
@@ -1320,7 +1322,8 @@ final void setParentController(@Nullable Controller controller) {
 
     private void ensureRequiredConstructor() {
         Constructor[] constructors = getClass().getConstructors();
-        if (getBundleConstructor(constructors) == null && getDefaultConstructor(constructors) == null) {
+        if (getBundleConstructor(constructors) == null && getDefaultConstructor(constructors) == null
+                && FACTORY instanceof ControllerFactory.DefaultControllerFactory) {
```

Comment:
If the user has installed a `ControllerFactory` we assume they know what they're doing. Maybe we should do some deeper/more expensive checking here? e.g. check that the installed `ControllerFactory` can create the `Controller`?